server/resolvers/TestPlanVersionOperations/updatePhaseResolver.js

```diff
@@ -253,6 +253,73 @@ const updatePhaseResolver = async (
         throw new Error('No test plan reports found.');
     }
 
+    // if (phase === 'CANDIDATE') {
```

Comment:
It seems this logic has been used elsewhere so you should be able to remove these commented out lines when ready.

Reply:
Good catch! I deleted this huge comment.